.github/workflows/machine_images.yaml

```diff
@@ -73,3 +73,8 @@ jobs:
           VERSION: ${{ env.PKR_VAR_coder_version }}
           MARKETPLACE_ACCESS_ROLE_ARN: ${{ secrets.MARKETPLACE_ACCESS_ROLE_ARN }}
           PRODUCT_IDENTIFIER: ${{ secrets.PRODUCT_IDENTIFIER }}
+
+      - name: Submit image to GCP Marketplace
+        run: ./github/workflows/gcp/submit_image.sh
+        env: 
+          VERSION: ${{ env.PKR_VAR_coder_version }}
```

Comment:
Where is this used? I don't see it used anywhere in `./github/workflows/gcp/submit_image.sh`

Reply:
it's not. we can remove

Reply:
Can I also remove this script and make it part of the main workflow?